github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -101,6 +101,35 @@ presubmits:
             memory: "4Gi"
         securityContext:
           runAsUser: 0
+  - name: pull-project-infra-test-bazel-config
+    run_if_changed: "github/ci/services/.*|external-plugins/.*|releng/.*|robots/.*|go.mod"
+    optional: false
```

Comment:
I think you are right, since this is the default. I was only wondering whether the actual behaviour might change when this value is present vs. when it's not. IIRC we talked about this, that we suspected something in those lines.

I can remove it anywway.

Reply:
Removed `optional: false`